stateMutators.js

```diff
@@ -114,16 +114,22 @@ export const removeFallingObject = (state, fallingObjId) => {
     }
 }
 
-// Object constructor for falling object
+/**
+ * Object constructor for falling object
+ *
+ * @param {number} columnIndex The index of the falling object column, there are 8 of them, index from 0 to 7
+ */
 export function FallingObject({
-    posX,
-    posY,
+    columnIndex,
+    yPos,
     numerator,
     denominator,
     id,
```

Comment:
Maybe this `id` field should be removed as these objects now are indexed by the `columnIndex` field?

Reply:
still don't know the purpose of this `id`, the `columnIndex` is not unique across `fallingObject` since more than one `fallingObject` can have the same `columnIndex`

Reply:
Okay so we still need this `id` field to identify a `fallingObject`, we need to find a way to generate this id so it is unique, maybe incremental?

Reply:
Yeah incremental sounds fine